armada-ths/ais
# 119 - Events - add contact email (Revised)

*Original pull request can be found [here](https://github.com/armada-ths/ais/pull/651).*

Changed the contact_email field for events to be based on a new separate text field rather than the contact person's default email. This is preferable because the contact person often had their KTH address associated with their user profile, meaning it is now possible to use any email address.

**Full testing steps from untouched ais:**

1. Apply database changes with ``python manage.py migrate``
2. Login as admin
3. /fairs/2019/events Click on one event
4. Edit the field "Contact e-mail" (close to bottom of page)
5. Save the event, make sure it responds OK
6. Add /signup to the end of the url
7. If it didn't work, try with /fairs/2019/events/31/signup (some events have no signup page - could also work with checking its "The event is published on the website"-box)
8. "Contact e-mail" should now be visible with the address you saved in step 4 and 5. It should have a "mailto"-link on it as well.
9. Finally, please confirm that the field is not visible on the signup page if it is left blank in the editing page of its event (i.e. remove the address entered in step 4 and save).
